pkg/services/control/types_test.go

```diff
@@ -133,11 +133,11 @@ func generateShardInfo() *control.ShardInfo {
 	uid, _ := uuid.NewRandom()
 	bin, _ := uid.MarshalBinary()
 
-	si.SetID(bin)
-	si.SetMode(control.ShardMode_READ_WRITE)
-	si.SetMetabasePath(path)
-	si.SetBlobstorPath(path)
-	si.SetWriteCachePath(path)
+	si.Shard_ID = bin
+	si.Mode = control.ShardMode_READ_WRITE
+	si.MetabasePath = path
+	si.BlobstorPath = path
+	si.WritecachePath = path
```

Comment:
Public fields instead of setter functions are dictated by protobuf or just a detail of generator implementation?

Reply:
Fixed.